src/Sarif.UnitTests/Visitors/RebaseUriVisitorTests.cs

```diff
@@ -158,7 +158,7 @@ public void RebaseUriVisitor_VisitFileData_PatchesParentUri()
                 {
                     new FileData { FileLocation = new FileLocation { Uri = rootfileUri, FileIndex = -0 }, ParentIndex = -1 },
                     new FileData { FileLocation = new FileLocation { Uri = childFileUri, FileIndex = 1 }, ParentIndex = 0 },
-                    new FileData { FileLocation = new FileLocation { Uri = childFileUri, FileIndex = -1 }, ParentIndex = -1 }
+                    new FileData { FileLocation = new FileLocation { Uri = childFileUri, FileIndex = 2 }, ParentIndex = -1 }
```

Comment:
Why did you add this line?

Reply:
I corrected the file index, which was wrong. have we addressed the matter of how the file index is populated for a file location within the files table itself? i.e., that it can either accurately reflect the actual index of its container or that it can be -1 (which isn't valid, the default, meaning it isn't expressed). or is this sufficiently clear in the spec, you think?

---
In reply to: [248386981](https://github.com/Microsoft/sarif-sdk/pull/1215#discussion_r248386981) [](ancestors = 248386981)

Reply:
Sorry, I misread the diff, didn't realize this line was already there.

---
In reply to: [248387734](https://github.com/Microsoft/sarif-sdk/pull/1215#discussion_r248387734) [](ancestors = 248387734,248386981)